test/remote/gateways/remote_hps_test.rb

```diff
@@ -261,4 +261,50 @@ def test_transcript_scrubbing
     assert_scrubbed(@credit_card.verification_value, transcript)
     assert_scrubbed(@gateway.options[:secret_api_key], transcript)
   end
+
+  def test_successful_purchase_with_apple_pay_raw_cryptogram_with_eci
+    credit_card = network_tokenization_credit_card('4242424242424242',
+      payment_cryptogram: 'EHuWW9PiBkWvqE5juRwDzAUFBAk=',
+      verification_value: nil,
+      eci: '05',
+      source: :apple_pay
+    )
+    assert response = @gateway.purchase(@amount, credit_card, @options)
+    assert_success response
+    assert_equal 'Success', response.message
+  end
+
+  def test_successful_purchase_with_apple_pay_raw_cryptogram_without_eci
+    credit_card = network_tokenization_credit_card('4242424242424242',
+      payment_cryptogram: 'EHuWW9PiBkWvqE5juRwDzAUFBAk=',
+      verification_value: nil,
+      source: :apple_pay
+    )
+    assert response = @gateway.purchase(@amount, credit_card, @options)
+    assert_success response
+    assert_equal 'Success', response.message
+  end
+
+  def test_successful_auth_with_apple_pay_raw_cryptogram_with_eci
```

Comment:
I had copied the test cases from the tests for Stripe. I can add an additional test for capturing an Apple Pay transaction, but it will perform a nearly identical capture as a normal open authorization.

Reply:
Cool. Just wanted to check if it provides additional coverage specific to this payment method. Sounds like that's not the case, I'm fine as is.